Symfony/src/AppBundle/Form/Type/RegistrationType.php

```diff
@@ -11,6 +11,9 @@
 
 class RegistrationType extends AbstractType
 {
+    /**
+     * @SuppressWarnings(PHPMD.UnusedLocalVariable)
```

Comment:
I think if you set it as an optional value `array $options = []` it won't complain. But if this is not the case, I'm good with the suppress in this case.